pkg/core/config/template.go

```diff
@@ -109,7 +109,7 @@ func (t *Template) readSecretsFiles() error {
 	return nil
 }
 
-// ReadFile reads an udpatecli values file, it can also read encrypted sops files
+// ReadFile reads an updatecli values file, it can also read encrypted sops files
```

Comment:
Ok, I lied. This is also a brand :) -- and there are a number of misspellings of it: https://github.com/updatecli/updatecli/pull/1330/commits/7bc2c3ab9f646e6c78e7123b2a5e0fc53a25e408

(This is not surprising, the words people use the most in a project are the ones most likely to be misspelled, and a project's name is likely to be used a lot...)

Reply:
That's a fair point